hack/verify-dirty.sh

```diff
@@ -29,6 +29,7 @@ echo "Done vendir sync!"
 # because the package-vendir-sync operation renders
 # the pakcage directory contents inaccessible to the
 # host machine user.
+GITHUB_ACTIONS="${GITHUB_ACTIONS:-}"
```

Comment:
Thanks for catching this. This can be done in line 34 as well:
`if [[ -n "${GITHUB_ACTIONS:-}" ]]; then`

Reply:
I like that suggestion. I'll put it in a commit.